dynawo/3rdParty/toolchain.sh

```diff
@@ -221,7 +217,7 @@ while (($#)); do
 			LIBARCHIVE_HOME=`echo $1 | sed -e 's/--libarchive-install-dir=//g'`
       ;;
     --gtest-install-dir=*)
-      GTEST_INSTALL_DIR=`echo $1 | sed -e 's/--gtest-install-dir=//g'`
```

Comment:
WHy is this modified here and not in libzip and libxml?

Reply:
It is actually not used the same way